osu.Game/Screens/Multi/Header.cs

```diff
@@ -0,0 +1,101 @@
+
+        private readonly OsuSpriteText screenTitle;
+        private readonly ScreenBreadcrumbControl breadcrumbs;
+
+        public Header(Screen initialScreen)
+        {
+            RelativeSizeAxes = Axes.X;
+            Height = HEIGHT;
+
+            Children = new Drawable[]
+            {
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = OsuColour.FromHex(@"2f2043"),
+                },
+                new Container
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Padding = new MarginPadding { Horizontal = SearchableListOverlay.WIDTH_PADDING },
+                    Children = new Drawable[]
+                    {
+                        new FillFlowContainer
+                        {
+                            Anchor = Anchor.CentreLeft,
+                            Origin = Anchor.BottomLeft,
+                            Position = new Vector2(-35f, 5f),
+                            AutoSizeAxes = Axes.Both,
+                            Direction = FillDirection.Horizontal,
+                            Spacing = new Vector2(10f, 0f),
+                            Children = new Drawable[]
+                            {
+                                new SpriteIcon
+                                {
+                                    Size = new Vector2(25),
+                                    Icon = FontAwesome.fa_osu_multi,
+                                },
+                                new FillFlowContainer
+                                {
+                                    AutoSizeAxes = Axes.Both,
+                                    Direction = FillDirection.Horizontal,
+                                    Children = new[]
+                                    {
+                                        new OsuSpriteText
+                                        {
+                                            Text = "multiplayer ",
+                                            TextSize = 25,
+                                        },
+                                        screenTitle = new OsuSpriteText
+                                        {
+                                            TextSize = 25,
+                                            Font = @"Exo2.0-Light",
+                                        },
+                                    },
+                                },
+                            },
+                        },
+                        breadcrumbs = new ScreenBreadcrumbControl(initialScreen)
+                        {
+                            Anchor = Anchor.BottomLeft,
+                            Origin = Anchor.BottomLeft,
+                            RelativeSizeAxes = Axes.X,
+                        },
+                    },
+                },
+            };
+
+            breadcrumbs.OnLoadComplete = d => breadcrumbs.AccentColour = Color4.White;
```

Comment:
shouldn't do this. needs to be subclassed instead

Reply:
Or should allow setting the colour prior to load().

Reply:
Probably. but framework so for another day...